vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -255,41 +256,52 @@ class OnboardingViewModel @AssistedInject constructor(
         }
     }
 
-    private fun handleRegisterAction(action: RegisterAction) {
+    private fun handleRegisterAction(action: RegisterAction, onNextRegistrationStepAction: (FlowResult) -> Unit) {
         currentJob = viewModelScope.launch {
             if (action.hasLoadingState()) {
                 setState { copy(isLoading = true) }
             }
-            runCatching { registrationActionHandler.handleRegisterAction(registrationWizard, action) }
```

Comment:
this diff in this function is a bit confusing, essentially this logic was extracted to a suspending `internalRegisterAction` which other `handle*` entry points can reuse